IdentityCore/src/util/NSURL+MSIDExtensions.m

```diff
@@ -198,7 +198,7 @@ - (NSURL *)msidURLWithQueryParameters:(NSDictionary *)queryParameters
     NSString *query = [components percentEncodedQuery];
     NSDictionary *queryParametersDictionary = [self msidQueryParameters];
 
-    for (NSString *key in [queryParameters allKeys])
+    for (NSString *key in [[queryParameters allKeys] sortedArrayUsingSelector: @selector(caseInsensitiveCompare:)])
```

Comment:
Is there any way to avoid this sorting? This function is executed often, and additional sorting is a performance hit.

Reply:
Reverted and updated test case to look for platform specific expected url